packages/evm/test/disputes/disputes-settle-round.js

```diff
@@ -80,13 +80,16 @@ contract('DisputeManager', ([_, drafter, appealMaker, appealTaker, juror500, jur
               assertBn(possibleRulings, 2, 'dispute possible rulings do not match')
               assertBn(finalRuling, expectedFinalRuling, 'dispute final ruling does not match')
 
-              const anotherReceipt = await court.executeRuling(disputeId)
+              const anotherReceipt = await court.rule(disputeId)
```

Comment:
Related to the comment above, I find this test acts weird. Should you be allowed to call `rule()` for the same disputeId multiple times on the IArbitrator? Are we expecting IArbitrables to implement logic to prevent calling this more than once?

Reply:
> Are we expecting IArbitrables to implement logic to prevent calling this more than once?

No

> Should you be allowed to call rule() for the same disputeId multiple times on the IArbitrator?

Yes, but note that the test is actually ensuring nothing happens (i.e. the event is not re-emitted, and the ruling stays the same)

Reply:
To come back to this, I think we actually want to say "Yes" to this question:

> Are we expecting IArbitrables to implement logic to prevent calling this more than once?

If an IArbitrable is only meant to receive a ruling once (e.g. to unlock some escrowed funds), then it is up to that contract to ensure it does not receive the same ruling twice.